configs/testing/metrics/dbscan.json

```diff
@@ -0,0 +1,174 @@
+{
+    "common": {
+        "lib": ["sklearn"],

```

Comment:
Why don't we just remove those brackets? We need these only when several values are going to be tested
As for me, this will only complicate the appearance